packages/dnb-eufemia/src/extensions/forms/Field/Option/OptionDocs.ts

```diff
@@ -12,6 +12,11 @@ export const OptionProperties: PropertiesTableProps = {
     type: ['string', 'React.Node'],
     status: 'optional',
   },
+  groupIndex: {
+    doc: `What group in the \`groups\` prop ([Field.Selection](/uilib/extensions/forms/base-fields/Selection/)) this item belongs to.`,
```

Comment:
```suggestion
    doc: `What group index in the \`groups\` prop ([Field.Selection](/uilib/extensions/forms/base-fields/Selection/)) this item belongs to.`,
```

Just to help clarify that it's not the actual group or group name that's expected, but rather the index of the group.